CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+- Bump greenmat from 3.5.1.2 to 3.5.1.3
```

Comment:
[memo]

## v3.5.1.3

* Remove support for ruby 2.5 and below
* Bump nokogiri from 1.6.0 to 1.11.7
  * Support cpu archtecture for arm64/aarch64 systems (like Apple's M1)
* Change ci platform to Github Actions